File: Scala/3.scala
object Reloj {
  def main(args: Array[String]): Unit = {
    print("\n________________CALCULAR HORA________________\n")
    print("Introduce los segundos: ")
    val datos : Int = scala.io.StdIn.readLine.toInt
    val horas: (Int) => Int = (x: Int) => x / 3600
    val minutos: (Int) => Int = (x: Int) => (x % 3600) / 60
    val segundos: (Int) => Int = (x: Int) => (x % 3600) % 60
    println("Hora:_"+horas(datos))
    println("Min:__"+minutos(datos))
    println("Seg:__"+segundos(datos))
  }
}

File: Python/3.py
def calcula_hora(segundos):
    segundos = int(segundos)
    time = []
    time.append(segundos//3600)
    time.append((segundos%3600)//60)
    time.append((segundos%3600)%60)
    return "HH:MM:SS -> " + str(time[0]) + ":" + str(time[1]) + ":" + str(time[2])


print(calcula_hora("10000"))